extensions/wikia/AdEngine/js/provider/gpt/helper.js

```diff
@@ -76,9 +76,8 @@ define('ext.wikia.adEngine.provider.gpt.helper', [
 				slotTargeting.rv = count.toString();
 			}
 		}
-		if (uapId) {
-			slotTargeting.uap = uapId.toString();
-		}
+
+		slotTargeting.uap = uapId ? uapId.toString() : 'none';
```

Comment:
Yes and yes.
(for some reasons gpt lib remembers what was set before and in order to update slot targeting parameter we need to set a new not-falsy value)
Do you know better way how to update UAP param on consecutive pageviews?